protobuf-delta-lake-sink/run-local.sh

```diff
@@ -19,17 +19,38 @@
 #   --partition-timestamp-column received_timestamp
 
 
-AWS_S3_ALLOW_UNSAFE_RENAME=true ../target/debug/protobuf-delta-lake-sink \
+# AWS_S3_ALLOW_UNSAFE_RENAME=true ../target/debug/protobuf-delta-lake-sink \
```

Comment:
Make separate bash files so it's clear which commands are still useful and which are outdated